lib/simple_segment/configuration.rb

```diff
@@ -5,14 +5,16 @@ class Configuration
     include SimpleSegment::Utils
     include SimpleSegment::Logging
 
-    attr_reader :write_key, :on_error, :stub, :logger
+    attr_reader :write_key, :on_error, :stub, :logger, :open_timeout, :read_timeout
```

Comment:
To make this a little more generic I think we could introduce a `http_options` setting, that accepts a hash. We can pas it directly to Net::HTTP so people could use any of the `opt` values from https://docs.ruby-lang.org/en/2.0.0/Net/HTTP.html#method-c-start

```ruby
SimpleSegment::Client.new(
  write_key: 'key',
  http_options: {
    open_timeout: 42,
    read_timeout: 42,
    close_on_empty_response: true,
    # ...
  }
)
```

We'd just need to make sure that `use_ssl` is true by default.

Reply:
That's a great suggestion, I just implemented it in 7176580